R/antigen.garnish_predict.R

```diff
@@ -223,7 +224,8 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
                                         "bitscore"))
 
     blastdt <- blastdt[, nmer := nmer %>% stringr::str_replace_all(pattern = "-", replacement = "")] %>%
-                  .[, WT.peptide := WT.peptide %>% stringr::str_replace_all(pattern = "-", replacement = "")]
+                  .[, WT.peptide := WT.peptide %>% stringr::str_replace_all(pattern = "-", replacement = "")] %>%
+                    .[!is.na(nmer) & !is.na(WT.peptide)]
```

Comment:
can these ever get returned as `""`, and would it be easier to white list a regular expression?

Reply:
I dont think they can be and we could whitelist, I dont know everything about blastp returns because it depends on the database you are using too.  So I'm not sure if it could ever be empty.  maybe an nchar != 0 check?

Reply:
ok lets leave as is

Reply:
Resolved with nchar check and whitelisted AAs after str_replace of "-|\\*" with "^[ARNDCQEGHILKMFPSTWYV]+$"